ArjunaJTS/pom.xml

```diff
@@ -22,6 +22,11 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>narayana-quickstarts-all</artifactId>
+    <groupId>org.jboss.narayana.quickstart</groupId>
+    <version>5.13.1.Final-SNAPSHOT</version>
+  </parent>
```

Comment:
Thanks @jmfinelli  for the review! the " <relativePath>../pom.xml</relativePath>" is the default for maven (and can be omitted). You have to specify it if the parent pom is not in the parent directory. Anyway if you think it would be clearer I can add it.

Reply:
Oh, I did not know about that! Thanks! You can consider this message as noise then ;-)